packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -195,37 +187,60 @@ const OpenQueueCard = ({
             </Tooltip>
           ))}
         </div>
-        {editingNotes && (
-          <SaveButton onClick={handleUpdate} size="large">
-            Save Changes
-          </SaveButton>
-        )}
-        {!editingNotes && (
-          <QueueCardButtonRow>
-            {isTA && (
-              <EditNotesButton
-                size="large"
-                onClick={() => {
-                  setEditingNotes(true);
-                }}
-              >
-                Edit Notes
-              </EditNotesButton>
-            )}
-            <Link
-              href="/course/[cid]/queue/[qid]"
-              as={`/course/${cid}/queue/${queue.id}`}
+        <OpenQueueCardDivider />
+        {editingNotes ? (
+          <div>
+            <NotesInput
+              defaultValue={queue.notes}
+              value={updatedNotes}
+              onChange={(e) => setUpdatedNotes(e.target.value as any)}
+            />
+          </div>
+        ) : queue.notes ? (
+          <div>
+            <Linkify
```

Comment:
out of curiosity what does this Linkify component do?

Reply:
i think this is for the react linkify library so people can click on links in the queue notes

Reply:
i think you and eddy worked on this lol

Reply:
OH u right LMAO my brain is ass